src/univers/version_range.py

```diff
@@ -729,6 +722,35 @@ def from_native(cls, string):
 
         return cls(constraints=constraints)
 
+    @classmethod
+    def sanitize_constraints(cls, string):
+        constraints = []
+        try:
+            specifiers = SpecifierSet(string)
+        except InvalidSpecifier as e:
+            raise InvalidVersionRange() from e
+        for spec in specifiers:
+            operator = spec.operator
+            version = spec.version
+            if "*" in version:
```

Comment:
Could we limit this to trailing "*" and not at any position?
I am not sure we can handle sanely other cases for now and the case with a star not at the end should likely be an exception.  
```suggestion
            if version.endswith("*"):
```